usercommandmgmt/usercommandmgmt.py

```diff
@@ -186,25 +193,138 @@ def get_highest_user_comm_allowance(self, member):
 
         return max(rel_usr_roles)
 
-    @staticmethod
-    def mod_evaluate_command(self, proposed_msg):
+    async def submit_for_approval(self, ctx, command, text):
         """
         posts the proposed command in the specified channel, and waits for a moderator's reaction to either
-        reject or accept the command. Will  return true if command_moderation is set to False or command
-        passes moderator evalutaion. False if it is rejected by a moderator.
-        Note: proposed_msg is the proposed command
+        reject or accept the command.
         """
-        if not command_moderation:
-            return True
-        else:
-            # submits command for evaluation and awaits response; TODO
-            pass
+        # create message to submit for mod approval
+        message_to_submit = (
+            "Sender: "
+            + ctx.author.name
+            + "\n"
+            + "Proposed Command: "
+            + "\n"
+            + command
+            + "\n"
+            + text
+            + "\n \n"
+            + ctx.message.jump_url
+        )
+
+        # finds the channel to send this message to: the moderator one; specified in configurable.py
+        mod_channel = self.find_channel_by_name(
+            mod_channel_name, ctx.message.guild.channels
+        )
+        # submit proposed command and relavant info to specified moderation channel
+        msg = await mod_channel.send(message_to_submit)
+        # adds reacts
+        await msg.add_reaction("✅")
+        await msg.add_reaction("❌")
+        self.create_comamnd_queue.append([msg, ctx, command, text])
+
+    def find_channel_by_name(self, channel_name, channel_List):
```

Comment:
Consider using discord.utils.get(..) and look up the channel name instead of creating a helper.